eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -72,6 +72,35 @@ jobs:
           eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
           eq(variables['isRollingBuild'], true))
 
+#
+# CoreCLR NativeAOT release build and additional libraries tests
+# Only when CoreCLR or library is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
```

Comment:
Just a question for my own understanding, could you have done Windows ARM here too, like in the [runtime.yml](https://github.com/dotnet/runtime/blob/218a4b1dad3f73eec8f9898761501c4cfa14698e/eng/pipelines/runtime.yml#L257), but we just chose x64 to get different coverage?

Reply:
I wanted to restrict the initial platforms to windows given the current GC issues in Linux